delivery_state/migrations/15.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Delete noupdate record because the xml_id is changed to
+    stock.mail_template_data_delivery_confirmation.
```

Comment:
More than changed is deprecated in favor of the other

Reply:
Changed! :)